MapboxNavigation/TopBannerViewController.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  TopBannerViewController.swift
+//  MapboxNavigation
+//
+//  Created by Jerrad Thramer on 1/11/18.
+//  Copyright © 2018 Mapbox. All rights reserved.
+//
+
+import UIKit
+import MapboxCoreNavigation
+import MapboxDirections
+
+class TopBannerViewController: UIViewController, StatusViewDelegate, InstructionsBannerViewDelegate {
+    //MARK: - Class Constants
+    static let lanesBackgroundColor: UIColor = UIColor(white: 247/255, alpha: 1.0)
+    static let statusBackgroundColor: UIColor = UIColor.black.withAlphaComponent(2/3)
+    
+    //MARK: - Outlets
+    @IBOutlet weak var instructions: InstructionsBannerView!
+    @IBOutlet weak var lanes: LanesView!
+    @IBOutlet weak var nextBanner: NextBannerView!
+    @IBOutlet weak var status: StatusView!
+    
+    //MARK: - Properties
+    var delegate: TopBannerViewControllerDelegate?
+    var isSimulatedNavigation: Bool = false
+    
+    //MARK: - View Lifecycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+        setBackgroundColors()
+    }
+    
+    // MARK: - Initial State
+    private func setupUI() {
+        instructions.delegate = self
+        status.delegate = self
+        [lanes, nextBanner, status].forEach { $0.isHidden = true }
+    }
+    
+    private func setBackgroundColors() {
+        instructions.backgroundColor = .white
+        lanes.backgroundColor = TopBannerViewController.lanesBackgroundColor
+        nextBanner.backgroundColor = .white
+        status.backgroundColor = TopBannerViewController.statusBackgroundColor
+    }
+
+    // MARK: - Model Updating
+    func update(progress: RouteProgress, location: CLLocation, secondsRemaining: TimeInterval) {
+        //update banner
+        instructions.update(progress: progress.currentLegProgress, location: location, secondsRemaining: secondsRemaining)
+        
+        //update lanes
+        if let upComingStep = progress.currentLegProgress?.upComingStep, !progress.currentLegProgress.userHasArrivedAtWaypoint {
+            updateLaneViews(step: upComingStep, durationRemaining: progress.currentLegProgress.currentStepProgress.durationRemaining)
```

Comment:
Investigate why one view uses the `secondsRemaining` argument and another view seems to pull the same value(?) out of `RouteProgress`.